templates/footer.twig

```diff
@@ -4,15 +4,12 @@
         {% include 'footer_country_selector.twig' %}
     </div>
     {% endif %}
-    <div class="container d-flex flex-column flex-lg-row-reverse">
```

Comment:
Remove the `flex-lg-row-reverse` bootstrap's class which was wrongly [merged a while ago](https://github.com/greenpeace/planet4-master-theme/commit/31b4b18359006498dd64dd405b5472e16e69facc#diff-9ccc5c1e49803dab74b7d5fb3b391bb54c4521688a0f01b887e1d082a4759e60R2).

Reply:
I think this was intentional, so that on mobile the social media links are shown first? (see [GPI's website](https://www.greenpeace.org/international/) for example)

Reply:
I agree! (also that is my code 🙃)

I've decided to change `flex-column flex-lg-row-reverse` with `flex-column-reverse flex-lg-row` to get the tab feature properly working on desktop.

Reply:
@dantovbein 
What about using the CSS [order](https://developer.mozilla.org/en-US/docs/Web/CSS/order) property here?
I think we could use it and avoid depending on the `switchOrders` function and a new listener that is always active.